lxd/events.go

```diff
@@ -2,6 +2,8 @@ package main
 
 import (
 	"fmt"
+	"github.com/lxc/lxd/lxd/project"
+	"github.com/pkg/errors"
 	"net/http"
 	"strings"
```

Comment:
We always sort our imports like:

```
built-in packages

external packages

internal packages
```

So in this case you should be changing this chunk of imports to:
```
    "fmt"
	"net/http"
	"strings"

	"github.com/pkg/errors"

	"github.com/lxc/lxd/lxd/db"
 	"github.com/lxc/lxd/lxd/project"
	"github.com/lxc/lxd/lxd/rbac"
	"github.com/lxc/lxd/lxd/response"
	"github.com/lxc/lxd/shared"
	"github.com/lxc/lxd/shared/logger"
```

Reply:
@tomponline I thought we were also deprecating the use of `pkg/errors`, what's the equivalent for `errors.Is` again?

Reply:
@stgraber yeah we are, although I thought it was already imported already, but as it wasn't we should use `errors` package and https://pkg.go.dev/errors#Is

Reply:
Apologies I thought I had fixed all of these.